platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/utils/TileLoadingMeasurementUtils.java

```diff
@@ -154,6 +162,37 @@ private static String getWindowSize() {
       int height = metrics.heightPixels;
       return "{" + width + "," + height + "}";
     }
+
+    private static String getWifiState() {
```

Comment:
wifi state might not be the most appropriate name as return types range from:
CONNECTION_WIFI, CONNECTION_CELLULAR and CONNECTION_NONE